### PR TITLE
Add function to transform before sending to ES

### PR DESCRIPTION
**Pull request by cdelgadob**
Add optional transformation function to manipulate records

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -46,6 +51,9 @@ exports.pushStream = async (
       case 'INSERT': {
         let body = converter(record.dynamodb.NewImage)
         body = removeEventData(body)
+        if (transformFunction) {
+          body = transformFunction(body)
```
Comment (matrus2) [inline src/index.js]: What do you think of passing `record.eventName` to this function?
`transformFunction(body, record.eventName)`
then you can manipulate based on `eventName`?

File: README.md [hunk 2]
```diff
@@ -60,6 +60,7 @@ Upload Lambda to AWS and _star_ this repository if it works as expected!!
 | index  | The name of ElasticSearch index (string). If not provided will set the same as DynamoDB table name | optional
 | type  | The type of the ElasticSearch document (string). If not provided will set the same as DynamoDB table name | optional
 | refresh  | Force ElasticSearch refresh its index immediately [more here](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-refresh.html) (boolean). Default: true | optional
+| transformFunction  | A function to transform each record before sending them to ES. Applies to INSERT and UPDATE operations | optional
```
Comment (matrus2) [inline README.md]: I would explain more deeply with the example

File: src/index.js (hunk 1 above)
Comment (cdelgadob) [reply src/index.js]: Well, you could easily do that, but I don't see the value: Unless I'm missing something, you either delete a record, in which case you will use the id, or create/update a record, in which case you need to do the same data transformations.
Please let me know if you have a different point of view.

File: README.md (hunk 2 above)
Comment (cdelgadob) [reply README.md]: Maybe improving the description would work. You can pass a function which return the record with the transformations. And then refer the user to the test files.

**Comment by matrus2** [comment]
Disregard my comments

**Pull request merged**